agents/mongodb/internal/profiler/profiler_test.go

```diff
@@ -207,20 +207,20 @@ func TestProfiler(t *testing.T) {
 		assert.Nil(t, err)
 
 		want := map[string]interface{}{
-			"indexFilterSet": bool(false),
+			"indexFilterSet": false,
+			"namespace":      "test_00.people",
 			"parsedQuery": map[string]interface{}{
 				"name_00�": map[string]interface{}{
```

Comment:
let's use `\xfdd` type vaues instead of `�`

Reply:
^

Reply:
@percona-csalguero could you try to replace it with `\ufffd`